packages/components/pie-modal/src/defs.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Modal heading levels/tags
+ */
+export enum HEADING_LEVELS {
```

Comment:
I will check before merging the PR if the union literal work is ready or not, if not, I reckon we shouldn't block the PR because of that and assume a clean up will needed 👍